gdpr_classifier/layers/llm/__init__.py

```
@@ -0,0 +1,7 @@
+"""LLM provider abstractions for Article9Layer and CombinationLayer."""
+
+from gdpr_classifier.layers.llm.gemini_provider import GeminiProvider
+from gdpr_classifier.layers.llm.ollama_provider import OllamaProvider
+from gdpr_classifier.layers.llm.provider import LLMProvider, LLMProviderError
+
+__all__ = ["LLMProvider", "LLMProviderError", "OllamaProvider", "GeminiProvider"]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)